_posts/2024-01-16-fido-passkey.md

```diff
@@ -0,0 +1,1127 @@
+                alert('Login failed.');
+            });
+        });
+    }
+}
+```
+
+Paste the following code in the `home.component.html` file.
+```html
+<div class="container mt-5">
+    <div class="row">
+        <div class="col-sm-12 col-md-8 info-text mt-5">
+            <p>Experience a new era of online security with FIDO Passwordless Authentication. Say goodbye to the hassle of remembering and managing passwords. 
+            Our cutting-edge technology ensures a seamless and secure login process for users, making online interactions effortless and worry-free.</p>
+        </div>
+
+        <div class="col-sm-12 col-md-4">
+            <form>
+                <div class="mb-3">
+                    <label for="username" class="form-label">Username</label>
+                    <input type="text" class="form-control" id="username" aria-describedby="usernameHelp"
+                           [(ngModel)]="username" [ngModelOptions]="{standalone: true}">
+                    <div id="usernameHelp" class="form-text">We'll never share your info with anyone else.</div>
+                </div>
+                <button type="submit" class="btn btn-primary" (click)="register()">Register</button>
+                <button type="submit" class="btn btn-primary ms-2" (click)="login()">Login</button>
+            </form>
+        </div>
+    </div>
+</div>
+```
+
+Paste the following code in the `user.component.html` file.
+```html
+<p>Welcome</p>
+```
+
+## Conclusion
+Embracing FIDO and Passkeys represents a leap forward in security and user-friendliness. 
+By shifting away from traditional password-based authentication methods, users benefit from heightened security measures while enjoying a more seamless and user-friendly experience. 
+The combination of FIDO's robust security protocols and the convenience of passkeys not only enhances protection against cyber threats but also simplifies the user authentication process, 
+contributing to a more secure and user-centric online environment. Embracing these advancements not only aligns with the evolving landscape of digital security 
+but also empowers users with a more efficient and trustworthy means of safeguarding their online identities. 
+It's a win-win that positions FIDO and passkeys as a compelling choice for the future of secure and user-friendly authentication.
+
+
+## Extra
+If you're interested in exploring the implementation details, you can access the frontend and backend code on GitHub.
+* [Backend](https://github.com/nicholasM95/passwordless-backend){:target="_blank" rel="noopener noreferrer"}
+* [Frontend](https://github.com/nicholasM95/passwordless-frontend){:target="_blank" rel="noopener noreferrer"}
+
+If you want to start the application locally, you can start the Docker containers and visit [the project locally](http://localhost:4200){:target="_blank" rel="noopener noreferrer"}.
+```bash
+docker run -p 8080:8080 nicholas95/passwordless-backend:v1
```

Comment:
Make sure to include in the readme that this is not production ready code (even if it is...) and that it's provided as is. 

And include a licence ;) Preferably MIT.